tabs-tabs-tabs/tabs.js

```diff
@@ -12,6 +12,30 @@ function firstUnpinnedTab(tabs) {
   }
 }
 
+function listTabs() {
+  chrome.tabs.query({
+    currentWindow: true
+  }, function(tabs) {
+    var tabsList = document.getElementById('tabs-list'),
+        currentTabs = document.createDocumentFragment();
+
+    tabsList.innerHTML = '';
+
+    for (var tab of tabs) {
+      if (!tab.active) {
+        var tabLink = document.createElement('a');
+
+        tabLink.innerHTML = tab.title || tab.id;
```

Comment:
Use of innerHTML is not recommended due to XSS and performance concerns, I think using tabLink.textContent will work here instead?